satpy/readers/avhrr_l1b_gaclac.py

```diff
@@ -80,16 +88,28 @@ def get_dataset(self, key, info):
                 data = self.reader.lats
             else:
                 data = self.reader.lons
+        elif key.name in ['sensor_zenith_angle', 'solar_zenith_angle',
+                          'sun_sensor_azimuth_difference_angle']:
+            sat_azi, sat_zenith, sun_azi, sun_zenith, rel_azi = self.reader.get_angles()
+            if key.name == 'sensor_zenith_angle':
+                data = sat_zenith
+            elif key.name == 'solar_zenith_angle':
+                data = sun_zenith
+            elif key.name == 'sun_sensor_azimuth_difference_angle':
+                data = rel_azi
         else:
             if self.channels is None:
                 self.channels = self.reader.get_calibrated_channels()
-
             data = self.channels[:, :, self.chn_dict[key.name]]
 
         chunk_cols = data.shape[1]
         chunk_lines = int((CHUNK_SIZE ** 2) / chunk_cols)
-        return xr.DataArray(da.from_array(data, chunks=(chunk_lines, chunk_cols)),
-                            dims=['y', 'x'], attrs=info)
+        res = xr.DataArray(da.from_array(data, chunks=(chunk_lines, chunk_cols)),
+                           dims=['y', 'x'], attrs=info)
```

Comment:
Adding the scanline timestamps as additional y-coordinate would be nice here. But since this PR seems urgent and the CF writer does not completely support timestamps (yet), we can do that later. Added it to my TODO List.

Reply:
I can try to add it tonight. Is the cf writer crashing if we add the times atm ?

Reply:
Done